### PR TITLE
integrated tab bar from react navigation

### PR DESCRIPTION
**Pull request by injas-bigbinary**
@sangameshsomawar _a

### CODE REVIEW

**Comment by injas-bigbinary** [comment]
https://user-images.githubusercontent.com/87293196/184067489-572152a7-5086-4091-bf6a-d35c03d92351.mp4

**Review by sangameshsomawar — COMMENTED** [review]
@injas-bigbinary Let me know your thoughts. I am thinking to make component lightweight and easily pluggable. Soon we will be building different topbar for neetoPlanner as well. So its better to keep just the custom tab bar component in the library and move all configs to the host app.


https://user-images.githubusercontent.com/93119254/184297709-4cd2a5d3-af22-4624-b895-3ac5e97eaf9b.mp4

**Comment by injas-bigbinary** [comment]
@sangameshsomawar Updated

File: src/components/SegmentedTopBar.js [hunk 1]
```diff
@@ -39,60 +40,54 @@ const width = Dimensions.get("screen").width - 32;
...
- *   const [tabIndex, setTabIndex] = useState(1);
  *
  *  return (
- *    <Container flex={1} alignItems="center" justifyContent="center">
- *      <Container my={3}>
- *        <SegmentPicker
- *          tabs={["On", "Off"]}
- *          currentIndex={tabIndex}
- *          onChange={setTabIndex}
+ *    <NavigationContainer>
```
Comment (sangameshsomawar) [inline src/components/SegmentedTopBar.js:54]: @injas-bigbinary import `NavigationContainer`,  creattion of `Tab` is missing in the example.

**Review by sangameshsomawar — APPROVED** [review]
@injas-bigbinary New implementation looks perfect. Posted a minor comment. @souravbaranwal Please integrate this in neetoInvoice on priority.

**Pull request merged**